### PR TITLE
Reviews controller

### PR DESCRIPTION
**Pull request by ajkatom**
created review controllers and routes

### CODE REVIEW

File: server/controllers/review.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  });
+});
+
+exports.getallreviewsforSitter = asyncHandler(async (req, res, next) => {
+  const reviews = Review.find({
+    sitterId: req.body.sitterId,
+  });
+  if (!reviews.length) {
+    res.json({ message: 'No reviews found be the first to write one!!' });
+  }
```
Comment (ezeilo-su) [inline server/controllers/review.js:28]: I don't see any need for this check. Let's just return the result and allow the client to decide how to render.

**Review by ezeilo-su — COMMENTED** [review]
Good work with the controller. I can see the code from the previous PR still show up here. You should always rebase your branch to reflect only the code pertaining to your current PR.

File: server/controllers/review.js (hunk 1 above)
Comment (ajkatom) [reply server/controllers/review.js:28]: i needed to branch off a previous because i needed the sitterId

**Review by ezeilo-su — COMMENTED** [review]
Thank you for making the requested changes.

**Pull request merged**